tests/unittests/utils/tabular/features/generators/test_label_encoder.py

```diff
@@ -0,0 +1,32 @@
+
+import numpy as np
+
+from autogluon.utils.tabular.features.generators import LabelEncoderFeatureGenerator
+
+
+def test_label_encoder_feature_generator(generator_helper, data_helper):
+    # Given
+    input_data = data_helper.generate_multi_feature_standard()
+
+    generator = LabelEncoderFeatureGenerator()
+
+    expected_feature_metadata_in_full = {
+        ('category', ()): ['cat'],
```

Comment:
what about test for 'object' dtype, or is that unnecessary?

Reply:
Unnecessary at the moment, unless we want to add support for both category and object for many of these cases. This means that LabelEncoder would have to have a pre_generator of CategoryFeatureGenerator to remain stateful, so it probably isn't ideal. Currently it does pass in object and (correctly) drops it.